.github/workflows/windows-periodic.yml

```diff
@@ -0,0 +1,169 @@
+# Workflow intended to periodically test containerd integration tests
+
+name: win-periodic-integration
+
+on:
+  schedule:
+  - cron: "0 */12 * * *"
+
+env:
+  AZURE_DEFAULT_LOCATION: westeurope
+  AZURE_SUBSCRIPTION_ID: ${{ secrets.AZURE_SUB_ID }}
+  AZURE_DEFAULT_VM_SIZE: Standard_D2s_v3
+  PASSWORD: Passw0rdAdmin # temp for testing, will be generated
+  DEFAULT_ADMIN_USERNAME: azureuser
+  SSH_OPTS: "-o ServerAliveInterval=20 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+  REMOTE_VM_BIN_PATH: "c:\\containerd\\bin"
+
+jobs:
+  winIntegration:
+    strategy:
+      matrix:
+        win_ver: [sac2004]
+        include:
+        - win_ver: sac2004
```

Comment:
How much work would it be to also add WS2019 to the matrix? That would be great to have coverage on as well.

Reply:
Very little, just adding the image & version naming to the matrix. I'll run a couple of test runs before changing, just to be sure it's ok & submit a amend to the PR.